### PR TITLE
test(components): google-cloud-sdk - Adding unit test infra and place holder.

### PR DESCRIPTION
**Pull request by SinaChavoshi**
Adding unit tests infra and place holders for Google Cloud SDK.

### CODE REVIEW

File: test/presubmit-components-google-cloud.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+source_root=$(pwd)
+
+cd "$source_root/components/google-cloud"
```
Comment (Bobgy) [inline test/presubmit-components-google-cloud.sh:18]: I'd suggest moving this script to this folder and call it  presubmit.sh, moving related things together generally helps maintenance

**Review by Bobgy — COMMENTED** [review]
Thank you for setting up test infra for GCP components! I always like to see more tests :)

File: test/presubmit-components-google-cloud.sh (hunk 1 above)
Comment (SinaChavoshi) [reply test/presubmit-components-google-cloud.sh:18]: Done, just to be sure you wanted me to move the `test/presubmit-components-google-cloud.sh` script to `components/google-cloud/` folder? if this is not correct kindly advise.

**Comment by Bobgy** [comment]
renamed title to conform to our PR title format

I don't have a lot of experience on python, the test infra setup looks good to me!
/LGTM

Do you need any review from others?

File: components/google-cloud/google_cloud_components/aiplatform/utils.py [hunk 2]
```diff
@@ -46,7 +46,7 @@ def make_args(sa):
     return "\n".join(additional_args)
 
   def component_yaml_generator(**kwargs):
-    """Funciton to create the actual component yaml for the input kwargs.""" 
+    """Funciton to create the actual component yaml for the input kwargs."""
```
Comment (sasha-gitg) [inline components/google-cloud/google_cloud_components/aiplatform/utils.py:49]:
```suggestion
    """Function to create the actual component yaml for the input kwargs."""
```

**Review by sasha-gitg — APPROVED** [review]

**Pull request merged**